beta-src/src/utils/map/WDArrowMarkerDefs.tsx

```diff
@@ -1,55 +1,94 @@
 import * as React from "react";
+import ArrowColor from "../../enums/ArrowColor";
 import ArrowType from "../../enums/ArrowType";
 import webDiplomacyTheme from "../../webDiplomacyTheme";
 
-const WDArrowMarkerDefs = function (): React.ReactElement[] {
-  return Object.entries(webDiplomacyTheme.palette.arrowColors).map(
-    ([arrowType, config]) => {
-      switch (arrowType) {
-        case ArrowType.MOVE_SUPPORT:
-          return (
-            <marker
-              id={`arrowHead__${arrowType}`}
-              markerWidth={8}
-              markerHeight={8}
-              refX="2%"
-              refY={4}
-              orient="auto"
-            >
-              <polygon
-                points="0 0, 8 4, 0 8, 0 7, 6 4, 0 1"
-                fill={config.main}
-              />
-            </marker>
-          );
-        case ArrowType.HOLD_SUPPORT:
-          return (
-            <marker
-              id={`arrowHead__${arrowType}`}
-              markerWidth={8}
-              markerHeight={8}
-              refX="2%"
-              refY={4}
-              orient="auto"
-            >
-              <polygon points="0 0, 0 8, 1 8, 1 0" fill={config.main} />
-            </marker>
-          );
-        default:
-          return (
-            <marker
-              id={`arrowHead__${arrowType}`}
-              markerWidth={8}
-              markerHeight={8}
-              refX={arrowType === ArrowType.FOREIGN_IMPLIED ? 0 : 7.1}
-              refY={4}
-              orient="auto"
-            >
-              <polygon points="0 0, 8 4, 0 8" fill={config.main} />
-            </marker>
-          );
-      }
-    },
+const WDArrowMarkerColors = function (
+  arrowType: ArrowType,
+): React.ReactElement {
+  switch (arrowType) {
+    case ArrowType.SUPPORT:
+      return (
+        <>
+          {Object.entries(webDiplomacyTheme.palette.arrowColors).map(
+            ([arrowColor, config]) => {
+              return (
+                <marker
+                  id={`arrowHead__${ArrowType[arrowType]}_${ArrowColor[arrowColor]}`}
+                  markerWidth={8}
+                  markerHeight={8}
+                  refX="2%"
+                  refY={4}
+                  orient="auto"
+                >
+                  <polygon
+                    points="0 0, 8 4, 0 8, 0 7, 6 4, 0 1"
+                    fill={config.main}
+                  />
+                </marker>
```

Comment:
you should be able to just do an implicit return here for all these markers in this file.
```
 => 
                <marker
                  id={`arrowHead__${ArrowType[arrowType]}_${ArrowColor[arrowColor]}`}
                  markerWidth={8}
                  markerHeight={8}
                  refX="2%"
                  refY={4}
                  orient="auto"
                >
                  <polygon
                    points="0 0, 8 4, 0 8, 0 7, 6 4, 0 1"
                    fill={config.main}
                  />
                </marker>
```